subprojects/shipkit/src/main/groovy/org/shipkit/internal/gradle/java/tasks/CreateDependencyInfoFile.java

```diff
@@ -1,74 +1,76 @@
 package org.shipkit.internal.gradle.java.tasks;
 
 import org.gradle.api.artifacts.*;
-import org.json.simple.JsonArray;
-import org.json.simple.JsonObject;
-import org.json.simple.Jsoner;
+import org.shipkit.internal.gradle.util.StringUtil;
 import org.shipkit.internal.notes.util.IOUtil;
 
-import java.util.ArrayList;
-import java.util.LinkedHashMap;
+import java.util.SortedSet;
+import java.util.TreeSet;
 
 public class CreateDependencyInfoFile {
 
-    private static final String DESCRIPTION = "This file was generated by Shipkit Gradle plugin. See http://shipkit.org.";
+    public static final String SEPARATOR = ":";
+    public static final String NEWLINE = "\r\n";
+    public static final String DEPENDENCY_INDENT = NEWLINE + " - ";
+    public static final String ARTIFACT_INDENT = NEWLINE + "   - ";
 
-    public void createDependencyInfoFile(CreateDependencyInfoFileTask task) {
-        JsonObject result = createJsonObject();
-        result.put("description", DESCRIPTION);
+    private static final String DESCRIPTION = "This file was generated by Shipkit Gradle plugin. " +
```

Comment:
Let's make the textual description as short as possible. It will be hard to change later on. It's better to link to our docs instead (I just added http://dependency-info.shipkit.org subdomain to point to our docs ;):

```
# Description
Generated by Shipkit - http://dependency-info.shipkit.org
```

Reply:
Cool, thanks for the subdomain, we can redirect to docs for this file later :)